include/zephyr/arch/x86/ia32/scripts/shared_kernel_pages.ld

```diff
@@ -48,6 +48,9 @@
 #endif /* CONFIG_X86_USERSPACE */
 	. += (GDT_NUM_ENTRIES + GDT_NUM_TLS_ENTRIES) * 8;
 #endif /* LINKER_ZEPHYR_FINAL */
+#if defined(CONFIG_SOC_FAMILY_INTEL_ISH) && defined(CONFIG_PM)
```

Comment:
There's no `CONFIG_SOC_FAMILY_INTEL_ISH` in the upstream tree. This patch should probably be combined with a PR that adds the SoC.

Reply:
Done. Changed to use generic config way to set the reserved GDT entries, please review.